README.md

```diff
@@ -1,60 +1,63 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
 
-> Description the project.
 
+# Gamers Rising Global Fevstival 2022
 
-## Built With
+> This is a version of an Imaginary website for a gaming festival I developed as part of my capstone project for the Microverse program module 1. The website is for an upcoming gaming festival for Gamers Rising, an extreme gaming gaming company that is having their first event in Africa here in Kenya.
+
+![screenshot](./images/screencapture-1.png)
 
-- Major languages
-- Frameworks
-- Technologies used
+## Built With
 
+- HTML
+- Pure CSS
+- Vanilla JavaScript
+- Github
+- LightHouse
+- Webhint
+- Stylelint
+- EsLint
 ## Live Demo (if available)
 
 [Live Demo Link](https://livedemo.com)
 
```

Comment:
- [ ] Please, consider fixing the link to your Demo version. As you can there is no possibility to check, if the project deployed via GH pages or not. 💯

Reply:
Nice job done. 👍